charts/woodpecker/charts/server/templates/secret.yaml

```diff
@@ -11,3 +11,18 @@ data:
   {{ $key }}: {{ $value | b64enc }}
   {{- end }}
 {{- end }}
+---
+{{- if .Values.createAgentSecret }}
+{{- $secret := lookup "v1" "Secret" .Release.Namespace "woodpecker-default-agent-secret" -}}
+  {{- if or (not $secret) (not (hasKey $secret.data "WOODPECKER_AGENT_SECRET")) }}
+# Secret or key doesn't exist and createAgentSecret: true -> create secret
```

Comment:
IMO this indentation looks strange, and I would keep it at the same level as the wrapped content.